### PR TITLE
use upper-case collection and namespace names in the int tests

### PR DESCRIPTION
**Pull request by ivansenic**
**What this PR does**:
Adds integration tests for the #343, by simply changing how collection and namespace names are generated.. 

I also refactored the tests and moved away from the `CqlEnabledBaseIntegrationTest`, as now we can create and drop keyspaces using the jsonapi directly.. 

I was also able to implement a workaround and do creation/dropping of namespaces/collections in `@AfterAll/@BeforeAll` methods.. This provides one big advantage -> ability to run a single test from the class.. This was not possible before as it was depending on the order of the executed methods..

Note that majority of changes in this PR is simply changing `keyspaceId.asInternal() -> namespaceName` and was executed using the IDE. The important changes are in the two base int test classes.

**Which issue(s) this PR fixes**:
Followup of #343

### CODE REVIEW

File: src/test/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResourceBaseIntegrationTest.java [hunk 1]
```diff
@@ -20,7 +20,9 @@
 
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public abstract class CollectionResourceBaseIntegrationTest extends CqlEnabledIntegrationTestBase {
```
Comment (ivansenic) [inline src/test/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResourceBaseIntegrationTest.java:22]: The namespace name is not easy to change as it depends on the `CqlEnabledIntegrationTestBase`.. I think we should go away from the CQL based tests, as now we have all the commands to run the tests purely using jsonapi commands.. Should I adapt that in this PR?

Comment (tatu-at-datastax) [reply src/test/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResourceBaseIntegrationTest.java:22]: If it's simple enough to do, yes, I think that makes sense.

File: src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractNamespaceIntegrationTestBase.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.stargate.sgv2.api.common.config.constants.HttpConstants;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.eclipse.microprofile.config.ConfigProvider;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```
Comment (ivansenic) [inline src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractNamespaceIntegrationTestBase.java:17]: This is the only thing that is not standard anymore and still annoys me.. I can remove this, but then I need to make the `createNamespace()` method `static` as well, meaning I need a way to reference a `namespace` as a static var.. But then all test would have the same namespace and that is not what we want.. 

Static methods are also not overriable, so I can not allow sub-classes to control the name as well..

I guess we must live with this..

**Review by tatu-at-datastax — APPROVED** [review]
LGTM

**Pull request merged**